### PR TITLE
Allow to change status context again

### PR DESCRIPTION
**Pull request by 6543**
regression of #561

### CODE REVIEW

**Comment by anbraten** [comment]
Why should a user be able to change it? I don't see any real benefit, so I would just remove that option completely.

**Comment by 6543** [comment]
-> edge-case: two woodpecker instances on same forge

File: cmd/server/flags.go [hunk 1]
```diff
@@ -209,6 +209,12 @@ var flags = []cli.Flag{
 	//
 	// remote parameters
 	//
+	&cli.StringFlag{
+		EnvVars: []string{"STATUS_CONTEXT", "WOODPECKER_GITHUB_CONTEXT", "WOODPECKER_GITEA_CONTEXT"},
```
Comment (anbraten) [inline cmd/server/flags.go:213]:
```suggestion
		EnvVars: []string{"WOODPECKER_STATUS_CONTEXT", "WOODPECKER_GITHUB_CONTEXT", "WOODPECKER_GITEA_CONTEXT"},
```

**Review by anbraten — COMMENTED** [review]
Please update docs to match this setup as well and add a deprecation notice to the migration guide so we can remove the `WOODPECKER_GITEA_CONTEXT` and `WOODPECKER_GITHUB_CONTEXT` in the next version.

**Review by anbraten — APPROVED** [review]

**Review by anbraten — APPROVED** [review]

**Pull request merged**